tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -551,6 +627,37 @@ where
     }
 }
 
+struct Timings {
+    accumulated: u64,
+    entered: Option<Instant>,
```

Comment:
Since we know the time the span was created, as well as the accumulated time the span was active for, we can also determine its total time (`entered.elapsed()` when the span closed) and the time it spent "idle" (alive but not entered). It might be useful to display these durations as well (perhaps optionally?). 

In particular, I think the idle time can be useful for async code in particular, since it represents the amount of time a future spent _waiting_ on IO/other tasks.

Reply:
Yes, excellent idea! This comes for free when collecting timings (well, one more Instant per span), as you note, so we should just add it to the output.